lgtm.yaml

```diff
@@ -3,6 +3,8 @@ path_classifiers:
     - scripts/docs.py
   test:
     - tests
+  queries:
+    - exclude: py/import-and-import-from
```

Comment:
This makes it so all unused imports are ignored. You want to remove this, and add comments on the individual lines themselves which were false positives.

![image](https://user-images.githubusercontent.com/5950433/79135282-f42c4e00-7d63-11ea-899b-fe5ec3efb6d4.png)

Reply:
@pdxjohnny this actually does not ignore all the unused imports, in LGTM there is an error which is raised when we use something like - 
` import pathlib
from pathlib import Path`
this error will be raised even when we use `Path`
LGTM asks us to do the above thing like this - 
`Remove the from xxx import yyy statement. Add yyy = xxx.yyy if required.`

Should I still make a comment on particular file or is it okay to exclude these type of errors.

Reply:
Ah yes. We still don't want to ignore those, we should be fixing them instead